### PR TITLE
CP-26401: Ported build from _oasis to jbuilder.

### PR DESCRIPTION
**Pull request by kc284**
Also, updated maintainers and re-indented.

Signed-off-by: Konstantina Chremmou <konstantina.chremmou@citrix.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-install: setup.data setup.ml
-	ocaml setup.ml -install
+# requires odoc
+doc:
+	jbuilder build @doc
 
-uninstall:
-	ocamlfind remove $(NAME)
+reindent:
+	git ls-files '*.ml*' '**/*.ml*' | xargs ocp-indent --syntax cstruct -i
```
Comment (mseri) [inline Makefile]: `'*.ml*'` is enough, `ls-files` looks for the files recursively in the folders

**Review by mseri — APPROVED** [review]
Feel free to merge or modify the `Makefile` and then merge. Does this require specfile changes? It probably will require updating some `jbuild/_oasis` file to use `xapi-plugin` instead of `xapi_plugin`

**Review by mseri — CHANGES_REQUESTED** [review]
Can you please fix the travis build as well?

**Comment by kc284** [comment]
I did search but didn't find any references to xapi_plugin, so to my best belief only xs-opam needs an update.

**Comment by mseri** [comment]
It is possible that nothing is using it

**Review by mseri — APPROVED** [review]

**Pull request merged**